cmd/verifydependencies/verifydependencies.go

```diff
@@ -72,23 +72,26 @@ func main() {
 
 	for _, dep := range externalDeps.Dependencies {
 		for _, refPath := range dep.RefPaths {
-			file, err := os.Open(refPath.Path)
-			if err != nil {
-				log.Fatalf("error opening file %v : %v", refPath.Path, err)
-			}
-			matcher := regexp.MustCompile(refPath.Match)
-			depFileScanner := bufio.NewScanner(file)
-			var found bool
-			for depFileScanner.Scan() {
-				line := depFileScanner.Text()
-				if matcher.MatchString(line) && strings.Contains(line, dep.Version) {
-					found = true
-					break
+			func() {
```

Comment:
nit: i would have declared a function for this instead of using a lambda.
/lgtm